### ISSUE TITLE
Validation of format on [s]printf

### ISSUE THREAD

**Issue by trejjam** [thecodingmachine/phpstan-safe-rule#3]
Hi, first thank for a great idea with the /safe package.
I found out that using functions [s]printf from Safe namespace make PHPStan\Rules\Functions\PrintfParametersRule not working (it does not validate a number of parameters anymore)

**Comment by moufmouf** [comment]
Indeed, since the function used is `Safe\sprintf` instead of `sprintf` alone, the rule is not triggered anymore.

I would need to reimplement the rule for `Safe\sprintf` in the Safe PHPStan extension.
I don't know if this is easy to do yet. I'll keep the issue open until I have some time to figure that out.

**Comment by shish** [comment]
Closing as [s]printf are now natively safe in PHP8, and no longer part of the Safe library
